Index: linux-4.14.206/sound/soc/codecs/Kconfig
===================================================================
--- linux-4.14.206.orig/sound/soc/codecs/Kconfig
+++ linux-4.14.206/sound/soc/codecs/Kconfig
@@ -1092,7 +1092,8 @@ config SND_SOC_WM8985
 	depends on SND_SOC_I2C_AND_SPI
 
 config SND_SOC_WM8988
-	tristate
+	tristate "Wolfson Microelectronics WM8988 codec driver"
+        depends on I2C
 
 config SND_SOC_WM8990
 	tristate
